horreum-web/src/domain/runs/DatasetComparison.tsx

```diff
@@ -213,28 +210,71 @@ type ViewComparisonProps = {
 
 function ViewComparison({headers, view, datasets, alerting}: ViewComparisonProps) {
     const [loading, setLoading] = useState(false)
-    const [rows, setRows] = useState<IRow[]>()
-    const token = useSelector(tokenSelector)
+    const [rows, setRows] = useState<IRow[]>([])
     useEffect(() => {
         setLoading(true)
         Promise.all(datasets.map(ds => datasetApi.getSummary(ds.id, view.id)))
             .then(
                 summaries => {
-                    setRows(
-                        view.components.map(vc => ({
-                            cells: [
-                                vc.headerName,
-                                ...summaries.map(summary => {
-                                    const render = renderValue(
-                                        vc.render,
-                                        vc.labels.length == 1 ? vc.labels[0] : undefined,
-                                        token
-                                    )
-                                    return render(summary.view?.[vc.id], summary)
-                                }),
-                            ],
-                        }))
-                    )
+                    const rows: any[][] = []
+                    summaries.forEach((summary, summary_index) => {
+                        let active_view = summary.view
+                        if (active_view != undefined) {
+                            view.components.forEach((component, component_index) => {
+                                let row = rows.find(r => r[0] === component.headerName)
+                                const pos = component_index + 1
+                                // @ts-ignore
+                                const active_component = active_view[pos]
+                                const ob = JSON.stringify( active_component )
+                                let beginning = null
+                                let val = null
+                                if ("\"" === ob.substring(ob.length -2, ob.length - 1)) {
+                                    beginning = ob.lastIndexOf(":", ob.length - 2)
+                                    val = ob.substring( beginning + 2, ob.length - 2)
+                                } else {
+                                    beginning = ob.lastIndexOf(":", ob.length - 1)
+                                    val = Number.parseFloat( ob.substring( beginning +1, ob.length - 1) )
```

Comment:
`ViewComponent` works slightly differently to `LabelValues` in that `ViewComponent` can have a value and a render JS function, therefore the `val` displayed to the user is not just a string or a float.

Previously there were calls to `renderValue` to create a render function `render()` that the was called to generate the value displayed to the user.

We need to retain this behaviour so that the data displayed in the UI is consistent